README.md

```
@@ -431,499 +73,144 @@ make run-example
-# Run standalone gRPC server (alternative to Docker)
-cvt serve --port 9550
+# Check deployment safety
+cvt can-i-deploy --schema my-api --version 2.0.0 --env prod
 ```
 
+See [CLI Reference](docs/reference/cli.mdx) for all commands.
+
 ## Project Structure
 
-```shell
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)